vscode/src/circuit.ts

```diff
@@ -229,7 +403,7 @@ function documentHtml(maybeUri: string, range?: IRange) {
     );
     const fsPath = escapeHtml(uri.fsPath);
     const lineColumn = range
-      ? escapeHtml(`:${range.start.line}:${range.start.character}`)
+      ? escapeHtml(`:${range.start.line + 1}:${range.start.character + 1}`)
```

Comment:
Do we need to pull this from the env? I can't remeber if it is just the debugger that has configurable offsets that we need to make calls for?

Reply:
Good question I hadn't thought about it, but in this case, no. `line`+`character` are coming from our rust debugger which consistently uses 0-indexing. Here we're adding 1 for display purposes, which consistently need to be 1-indexed for the user. So we're good.